index.html

```diff
@@ -50,6 +50,12 @@
             name: "Dave Longley", url: "https://digitalbazaar.com/",
             company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/",
             note: "2014-2022", w3cid: 48025
+          }, {
+            name: "Greg Bernstein", url: "https://www.grotto-networking.com/",
+            company: "Invited Expert", w3cid: 0
```

Comment:
@Wind4Greg, I need you to fill out your w3cid here. You can find out what it is by going here: 

https://www.w3.org/users/myprofile/

The URL will change into something that looks like this:

https://www.w3.org/users/41758/

Whatever number shows up in the URL is your w3cid.